keycloak/themes/hmda/login/template.ftl

```diff
@@ -60,6 +60,7 @@
               <div class="usa-alert-body">
                 <h3 class="usa-alert-heading">${message.type} Status</h3>
                 <p class="usa-alert-text">${message.summary}</p>
+                <p>For help account-related issues, please contact <a href="mailto:${properties.supportEmail!}">${properties.supportEmail}</a></em>.</p>
```

Comment:
Did you see my [comment](https://github.com/cfpb/hmda-platform-auth/pull/38#discussion_r91761456)? Others think having it is a good idea. Others being FEWDs and UI/UX folk.